docs/API.md

```diff
@@ -89,6 +89,41 @@ mapProps(
 
 Accepts a function that maps owner props to a new collection of props that are passed to the base component.
 
+This allows props to be derived from other props in a similar way to [reselect](https://github.com/reactjs/reselect).
+```js
+const User = ({ name, status }) =>
```

Comment:
IMO examples which are not fit to screen is unreadable ;-)
mapProps is so simple, is it needed so big example for it?